### PR TITLE
2024 champions cohort page & past cohorts 

### PR DESCRIPTION
**Pull request by stefaniebutland**
addressing https://github.com/nmfs-openscapes/nmfs-openscapes.github.io/issues/62

### CODE REVIEW

**Comment by eeholmes** [comment]
@stefaniebutland @jules32 I made a first stab at some text for the 2024 cohort page using some of the email text from the email announcement.

**Comment by stefaniebutland** [comment]
Noting an error & solution here. Pushing commits of text and 2 zoomie images of 1MB each (not sure why this gave the following error, but the solution allowed me to push):

```
>>> /usr/local/bin/git push origin HEAD:refs/heads/2024-champs-page
error: RPC failed; HTTP 400 curl 22 The requested URL returned error: 400
send-pack: unexpected disconnect while reading sideband packet
fatal: the remote end hung up unexpectedly
```

[Stackoverflow](https://stackoverflow.com/questions/62753648/rpc-failed-http-400-curl-22-the-requested-url-returned-error-400-bad-request) says
> I faced this error on my mac and solved with this command `git config --global http.postBuffer 157286400`.

and [thewayeye.net](https://thewayeye.net/posts/fixing-the-rpc-failed-http-400-error-in-git/) has some explanation.

> By default, Git’s buffer for HTTP operations is quite small. When pushing large files or numerous files simultaneously, the buffer can become overwhelmed, leading to the HTTP 400 error. Increasing the buffer size to 157286400 (approximately 150 MB) allows Git to handle larger payloads without running into this limitation. The above command increases the buffer size for all repositories on your system.

File: cohorts/NMFS-fall2024.qmd [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+For the fall 2024, the Openscapes Champions Cohorts had 190 applications for 120 spots. Participants were chosen to ensure equitable participation across science centers and regional and headquarter offices, with a preference for offices, teams and individuals that had not participated before and for teams involved in IRA supported projects. All six NOAA Fisheries science centers, Greater Atlantic Regional Fisheries Office, Alaska Regional Office, Office of Protected Resources, and Office of Sustainable Fisheries participated in the 2024 fall cohorts.
+
+This [cohort training](https://nmfs-openscapes.github.io/2024-nmfs-champions/) is taking place over 5 remote calls from October 8 to December 4, 2024. Cohort calls will cover:
+
+1)  setting the open science mindset and discussing psychological safety,
```
Comment (jules32) [inline cohorts/NMFS-fall2024.qmd:26]:
```suggestion
1)  setting the open science mindset and seeing peer stories of what this looks like,
```

File: cohorts/NMFS-fall2024.qmd [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+2)  an introduction to publishing and project management on GitHub,
+3)  instituting team culture and data strategies,
+4)  developing open communities and coding strategies, including Cloud, and
+5)  preparing a team pathway document to guide their open data science journey after the training.
+
+Champions learned from stories shared by past participants including the Alaska Marine Mammal Stock Assessment Report process ([recording](https://www.youtube.com/watch?v=wqbwcCXbFL8)) and the California Current Ecosystem Status Report Automation team ([recording](https://youtu.be/St_j0D-u7UU)). [Mentors from across NMFS](../mentors/index.qmd) attended some Cohort Calls and Coworking sessions to share their expertise and experience.
```
Comment (jules32) [inline cohorts/NMFS-fall2024.qmd:32]:
```suggestion
Champions learned from stories shared by past participants including the Alaska Marine Mammal Stock Assessment Report process ([recording](https://www.youtube.com/watch?v=wqbwcCXbFL8)) and the California Current Ecosystem Status Report Automation team ([recording](https://youtu.be/St_j0D-u7UU)). [Mentors from across NMFS](../mentors/index.qmd) supported Cohort Calls, Coworking sessions, and asynchronous chat channels to share their expertise and experience.
```

**Review by jules32 — APPROVED** [review]
some small edits

**Comment by jules32** [comment]
This looks great Stef and Eli! Ready for merging; I've added the photo to the folder. I'll message the comms team after it's online later today

**Pull request merged**